.cncc.style

```diff
@@ -0,0 +1,14 @@
+# Kind-specific patterns to check AST nodes against. Both python-clang and
+# libclang docs explain CursorKind, with differences in detail. See also:
+#  - https://github.com/llvm-mirror/clang/blob/aca4fe314a55cacae29e1548cb7bfd2119c6df4c/bindings/python/clang/cindex.py#L599
+#  - http://clang.llvm.org/doxygen/group__CINDEX.html#gaaccc432245b4cd9f2d470913f9ef0013
+#  - https://docs.python.org/2/library/re.html#regular-expression-syntax
+
+class_decl: '^([A-Z][a-z]+)+$'
+struct_decl: '^([A-Z][a-z]+)+$'
+field_decl: '^[a-z_]+$'
+var_decl: '^[a-z]+[a-z0-9_]*$'
+parm_decl: '^[a-z]*[a-z0-9_]*$'
+namespace: '^[a-z_]*$'
+cxx_method: '^([A-Z][a-z]+)+$'
+function_decl: '^[a-z]+([A-Z][a-z]+)*$'
```

Comment:
Can and should namespaces and field decls start with `_` and contain `_`? If field decls can start with `_` I think it should be stricter like `_?[a-z]`.